templates/taxi/driver_list.html

```diff
@@ -6,8 +6,13 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if user.id == driver.id %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that the `driver.username` field is available and correctly populated in the `Driver` model. If `username` is not a field in the `Driver` model, adjust the template to use the correct field.